pages/index.tsx

```diff
@@ -1,21 +1,19 @@
-import type { NextPage, GetServerSidePropsContext } from "next";
-import { unstable_getServerSession } from "next-auth/next";
-import { authOptions } from "pages/api/auth/[...nextauth]";
+import type { NextPage } from "next";
 import HomeLayout from "src/components/HomeLayout";
 import clsx from "clsx";
 import styles from "styles/index.module.css";
 
 import AppButton from "components/AppButton";
 
-const customH2 = "text-2xl lg:text-[2rem] lg:leading-[2.8rem] font-medium font-unbounded";
-const customP = "text-base lg:text-[1.25rem] lg:leading-[2rem] font-medium";
-
-const customGlobeVertPad = "mt-8 gap-8";
-
-const marginSubHeader = "mb-[24px] lg:mb-[40px]";
-const customBaseCard = "rounded-[57px] bg-white";
-const cardPadding = "p-[3rem] lg:p-[3.75rem]";
-const customBaseInnerCard = "rounded-[30px] border border-[#c6cdd0] p-6 md:p-8";
+const twStyles = {
```

Comment:
if you create object, so you can call it `customTwStyles` and remove prefix `custom` from fields

Reply:
Resolved in 4521dbf